lib/compute/auth-config.ts

```diff
@@ -103,14 +125,20 @@ export class AuthConfig {
             pattern: '.*',
             permissions: AuthConfig.readOnlyRolePermissions,
           },
-
           ],
         },
       },
     };
 
     jenkinsYaml.jenkins.authorizationStrategy = rolesAndPermissions;
-
+    if (typeof fineGrainedAccessItems !== 'undefined') {
```

Comment:
Does `!== undefined` better than `=== true` or similar?

Reply:
Both have different purpose. One checks if value is undefined and other one checks if value is truly assigned to the variable.